csharp/README.md

```diff
@@ -0,0 +1,28 @@
+# OpenTracing Tutorial - C#
+
+## Installing
```

Comment:
Maybe this should say something about setting up the build environment. Does it require running Visual Studio? Can people download it freely these days?  Would be good to avoid the dependency on VS if possible, and only use build from command line.

Reply:
It has no dependency on VS. You only need the .NET SDK that is either bundled with VS or can be downloaded from Microsoft. The examples can also be written just in notepad. The commands shown (dotnet run) are also just for command line. So no IDE dependency in the tutorial at all :)

Reply:
I added an IDE notice and other instructions.